zapcore/json_encoder_test.go

```diff
@@ -100,3 +100,17 @@ func TestJSONEncodeEntry(t *testing.T) {
 		)
 	})
 }
+
+func TestJSONEncodeEntryClosesNamespaces(t *testing.T) {
```

Comment:
The previous test doesn't actually write an Entry, since that simplifies most of the assertions.